express-crud/server.ts

```diff
@@ -0,0 +1,135 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const { grades } = req.body;
+    if (!grades) throw new ClientError(400, 'please provide grades');
+    const sql = `
+select *
+from "grades"
+where "grades" = $1
```

Comment:
There is not a column named `grades` in the table, so this should give you a SQL error when you try to run it.

What happens when you run `http :8080/api/grades`? It should return an array with 4 grades in it.